main/guides/agoric-cli/agd-query-tx.md

```diff
@@ -0,0 +1,167 @@
+# Using `agd` to make queries and transactions
+
+`agd` is the Agoric Cosmos App, analagous to `simd` in the [Cosmos SDK](https://docs.cosmos.network/) simapp or `gaiad` in
+the Cosmos hub. Most of the `simd` [query commands](https://docs.cosmos.network/v0.46/core/cli.html#query-commands) and [transaction commands](https://docs.cosmos.network/v0.46/core/cli.html#transaction-commands) work similarly in `agd`.
+
+::: tip agd for Building Dapps
+
+This section focusses on commands relevant to developing and deploying smart contracts and dapps. See also:
+
+- [Validators topics \- Agoric Community Forum](https://community.agoric.com/c/validators/9)
+- [Governance topics \- Agoric Community Forum](https://community.agoric.com/c/governance/6)
+- [Delegator Guide \(CLI\) \| Cosmos Hub](https://hub.cosmos.network/main/delegators/delegator-guide-cli.html)
+
+:::
+
+::: tip Installing agd
+
+Options include:
+
+- Use the [basic dapp local chain](../getting-started/#starting-a-local-agoric-blockchain) docker container: to run `agd status`, enter `yarn docker:bash` followed by `agd status`; or use `docker-compose exec agd agd status`.
+- Install from an [agoric-sdk release](https://github.com/Agoric/agoric-sdk/releases).
+
+:::
+
+If we invoke `agd` without arguments, it prints a list of available commands including:
+
+```
+Available Commands:
+
+  help                Help about any command
+  keys                Manage your application's keys
+  query               Querying subcommands
+  status              Query remote node for status
+  tx                  Transactions subcommands
+  version             Print the application binary version information
+
+Flags:
+  -h, --help                help for agd
+      --home string         directory for config and data (default $HOME)
+```
+
+## Query Commands
+
+In most cases, `agd query ...` is followed by a module name such as `bank`. An exception is `agd status`:
+
+### agd status
+
+Query remote node for status
+
+Example:
+
+```console
+$ agd status
```

Comment:
Are you suggesting that we inform our readers about the `jq` technique?

Or that the output currently in the docs is less readable than it should be?

Reply:
Yeah I'm suggesting we inform our readers about the `jq` technique, because the output of this command is a json blob without any formatting or whitespaces in between.